.github/workflows/widget_wave.yml

```
@@ -16,6 +16,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        # super important if you want to see all results, even if one fails.
+        fail-fast: false
```

actionlint (1.7.4)

expecting a single ${{...}} expression or array value for matrix variations, but found plain text node

(syntax-check)